pipeline_dsl/resources/cron.py

```diff
@@ -1,8 +1,12 @@
-class Cron:
-    def __init__(self, definition):
+from pipeline_dsl.resources.resource import AbstractResource, ConcourseResource
+from typing import Optional, Dict
+
+
+class Cron(AbstractResource):
```

Comment:
```suggestion
class Cron(AbstractResource[None]):
```

Reply:
Good point. I fixed it in https://github.com/SAP/pipeline-dsl/pull/16/commits/f57e178bab3eee96fab194f35bbe1bc7788bac85.